requirements.txt

```diff
@@ -1,2 +1,3 @@
 # These are the main requirements, but they have several other dependencies
 Red-DiscordBot==3.4.0
+profanity-chec==1.0.3
```

Comment:
Misspelled module name.

Reply:
Resolved this myself when I updated the minimum requirement for Red-DiscordBot as it caused a merge conflict.